recipes/game-networking-sockets/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class GameNetworkingSocketsConan(ConanFile):
+    name = "game-networking-sockets"
```

Comment:
[If we want to be as close to the upstream name as possible](https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#should-reference-names-use---or-_), the name should be `gamenetworkingsockets`

But I'm uncertain about this one. Maybe other reviews can give us their opionion